pkg/continuoustest/write_read_series.go

```diff
@@ -94,35 +94,51 @@ func (t *WriteReadSeriesTest) Init(ctx context.Context, now time.Time) error {
 }
 
 // Run implements Test.
-func (t *WriteReadSeriesTest) Run(ctx context.Context, now time.Time) {
+func (t *WriteReadSeriesTest) Run(ctx context.Context, now time.Time) error {
 	// Configure the rate limiter to send a sample for each series per second. At startup, this test may catch up
 	// with previous missing writes: this rate limit reduces the chances to hit the ingestion limit on Mimir side.
 	writeLimiter := rate.NewLimiter(rate.Limit(t.cfg.NumSeries), t.cfg.NumSeries)
 
+	// collect all errors on this test run
+	errs := []error{}
```

Comment:
Please use `multierror.MultiError` from `dskit` instead.

Reply:
Thanks! I was wondering if we already had a MultiError type. I didn't find this one, but I will update to use it.